demonstrations/learning2learn.py

```diff
@@ -0,0 +1,1205 @@
+
+.. meta::
+    :property="og:description": Use a classical recurrent neural network to initilize the parameters of a variational quatum algorithm.
+    :property="og:image": ../demonstrations/learning2learn/thumbnail.png
+
+.. related::
+
+   tutorial_qaoa_intro QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut problem
+
+
+In this tutorial we recreate the architecture proposed by Verdon et
+al. in the paper *Learning to learn with quantum neural networks via
+classical neural networks* [#l2l]_, using **PennyLane** and **TensorFlow**.
+In this paper, classical recurrent neural networks are used to assist
+the optimization of variational quantum algorithms.
+
+We will start with a brief theoretical overview explaining the problem
+and the setup used to solve it. After that, we will deep dive into the
+code to build a fully functioning model, ready to be further developed
+or customized for your own needs. Without further ado, let’s start!
+
+
+Problem: Optimization of Variational Quantum Algorithms (VQAs)
+------------------------------------------------------------------
+
+Recently, a big effort by the quantum computing community has been
+devoted to the study of variational quantum algorithms (VQAs, for short)
+which leverage quantum circuits with fixed shape but with tunable
+parameters to solve a desired target task. The idea is similar to
+classical neural networks, where the weights of the network are
+optimized during training to perform best in regression or
+classification tasks. Similarly, once the shape of the variational quantum
+circuit is chosen — something that is very difficult and sensitive to
+the particular task at hand — its tunable parameters are optimized
+iteratively by minimizing a cost (or loss) function, which measures how
+good the quantum algorithm is performing (see [#vqas]_ for a
+thorough overview on VQAs).
+
+A major challenge for VQAs relates to the optimization of tunable
+parameters, which was shown to be a very hard task to perform (a
+possible challenge is posed by the so called *barren plateaus*). In
+particular, parameter initialization plays a key role in this scenario,
+since initializing the parameters in the proximity of an optimal
+solution leads to faster convergence and better results with respect to
+random initialization, which could bring the model in a region in
+parameter space far from any good solution. Thus, a good initialization
+strategy is crucial to promote the convergence of local optimizers to
+local extrema and to select reasonably good local minima. Here, by local
+optimizer, we mean a procedure that moves from one solution to another
+by small (local) changes in parameter space. These are opposed to global
+search methods, which take into account the whole (or at least a large
+section) parameter space to propose a new solution.
+
+One such strategy could come from the classical machine learning
+literature.
+
+Solution: (Classical) Recurrent Neural Networks
```

Comment:
```suggestion
Solution: Classical Recurrent Neural Networks
```
Or simply remove "classical"

Reply:
Mmhh...Yeah, that "Classical" may be redundant. In my mind, it was used to stress that this is not a quantum generalization of a RNN, but just a purely classical one applied to a quantum problem.